Bluepill-runner/Bluepill-runner/BPRunner.m

```diff
@@ -274,7 +274,9 @@ - (int)runWithBPXCTestFiles:(NSArray<BPXCTestFile *> *)xcTestFiles {
             [BPUtils printInfo:INFO withString:@"%lu Simulator%s still running. [%@]",
              launchedTasks, launchedTasks == 1 ? "" : "s", listString];
             [BPUtils printInfo:INFO withString:@"Using %d of %d processes.", numprocs(), maxProcs];
-
+            if (numprocs() > maxProcs/2) {
```

Comment:
1) It should be at least 3/4
2) space before & after /
3) This number should be a constant

Reply:
updated